certz/certz.proto

```diff
@@ -26,33 +26,68 @@ option (gnoi.types.gnoi_version) = "0.2.0";
 
 // The Certificate Management Service exported by targets.
 //
-// This service exports one RPCs, Rotate(), which is used to rotate an existing
+// This service exports multiple RPCs. Three of them are used to manage SSL
+// profiles, one to check if the target has the capability to create a CSR and
+// one, Rotate(), which is used to rotate (i.e. updatde or replace) an existing
 // certificate, an existing trust bundle and/or an existing certificate
-// revocation list bundle on a target.
+// revocation list bundle that are part of a SSL profile on a target.
 //
 // Note that for the sake of data consistency and API simplicity, only one RPC
 // may be in progress at a time. An attempt to initiate another RPC while
 // one is already in progress will be rejected with an error.
 //
-// The service makes an assumption that the certificates it manages are
+// The service makes an assumption that the credentials it manages are
 // organized in the following _logical_ topology.
 //
 // Target (as seen from gNSI.certificate microservice point of view)
 // |
-// +-+ certificate (used by all gRPC services)
-// | +- certificate (with public key)
-// | +- private key
+// +-+ SSL profile for gNXI; always present; ssl_profile_id := "gNxI"
+// | |
+// | +-+ certificate
+// | | +- certificate (with public key)
+// | | +- private key
+// | |
+// | +-+ trust bundle (Certificate Authority certificates)
+// | | +- CA Root certificate
+// | | +- CA Intermediate Certificate
+// | |
+// | +-+ Certificate Revocation List bundle
+// |   +-+ Certificate Revocalion List A
+// |   +-+ Certificate Revocalion List B
 // |
-// +-+ trust bundle (Certificate Authority certificates)
-// | +- CA Root certificate
-// | +- CA Intermediate Certificate
-// |
-// +-+ Certificate Revocation List bundle
-//   +-+ Certificate Revocalion List A
-//   +-+ Certificate Revocalion List B
+// +-+ Another SSL profile used by another gRPC-based service
```

Comment:
Should we say here that this doesn't need to be a gRPC-based service? e.g., I think https server credentials could be provided here, right?

Reply:
Yes. I have removed `gRPC-based`